### PR TITLE
Martincivan/feat containerization

### PR DESCRIPTION
**Pull request by martincivan**
ECR already applied

### CODE REVIEW

File: lib/Log.php [hunk 1]
```diff
@@ -87,7 +88,8 @@ private static function createLogger(LogConfig $config) {
             $config->getContextPrefix()
         );
 
-        $handler = new FileHandler($config->getPath(), $config->getLevel());
+        //$handler = new FileHandler($config->getPath(), $config->getLevel());
+        $handler = new StreamHandler($config->getLevel());
```
Comment (matuszeman) [inline lib/Log.php:92]: Configurable?

File: devops/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# 
+
+```
+docker compose
```
Comment (matuszeman) [inline devops/README.md:4]: Remove or make complete.

File: .github/workflows/release_build.yml [hunk 3]
```diff
@@ -27,4 +27,33 @@ jobs:
       - name: Github Release
         uses: softprops/action-gh-release@v1
         with:
-          files: "./build/*.zip"
\ No newline at end of file
+          files: "./build/*.zip"
+  build-Docker:
```
Comment (matuszeman) [inline .github/workflows/release_build.yml:31]: Create workflow which would be able to build the image from any branch using workflow_dispatch event, so we can test changes from any branch.

File: devops/containers/app/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+# DEVELOMENT
+# ==========================================
+FROM prod-runtime AS dev
+
+RUN pecl install \
+    xdebug
+RUN docker-php-ext-enable \
+    xdebug
+
+ARG APM_AGENT_VERSION=1.9.1
```
Comment (matuszeman) [inline devops/containers/app/Dockerfile:71]: APM can be removed for now.

File: devops/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    volumes:
+      - ../:/app:ro
+      # App config
+      - ./deploy/containers/app/config/app-config.yml:/app-config.yml:ro
+      - ./deploy/containers/app/secrets/app-secrets:/app-secrets:ro
+      # PHP
+      - ./deploy/containers/app/config/usr--local--etc--php--conf.d--zzz-env.ini:/usr/local/etc/php/conf.d/zzz-env.ini:ro
+    environment:
+      # DEV
+      ELASTIC_APM_ENABLED: 1
```
Comment (matuszeman) [inline devops/docker-compose.yml:20]: Remove APM related config.

File: devops/ecr/main.tf [hunk 6]
```diff
@@ -0,0 +1,19 @@
+module "app-dev-ctx" {
```
Comment (matuszeman) [inline devops/ecr/main.tf:1]: I'd rename this folder to `resources` ... this folder could be used to provision any development related resource, not repositories only.

File: .github/workflows/release_build.yml [hunk 7]
```diff
@@ -27,4 +27,33 @@ jobs:
...
+    runs-on: ubuntu-latest
+    steps:
+      - id: meta
+        name: "Create image meta"
+        shell: bash
+        run: |
+          echo "image_tag=${GITHUB_REF_NAME##*/}" >> $GITHUB_OUTPUT
+          echo "image_repository=e7c0y4g3/la-resqu-app" >> $GITHUB_OUTPUT
```
Comment (matuszeman) [inline .github/workflows/release_build.yml:39]: This is: `e7c0y4g3/resqu-app`

**Review by matuszeman — APPROVED** [review]

File: .github/workflows/release_build.yml (hunk 3 above)
Comment (martincivan) [reply .github/workflows/release_build.yml:32]: added repo dispatch, but tag could be pushed anywhere

File: devops/containers/app/Dockerfile (hunk 4 above)
Comment (martincivan) [reply devops/containers/app/Dockerfile]: removed

File: devops/ecr/main.tf (hunk 6 above)
Comment (martincivan) [reply devops/ecr/main.tf]: renamed

File: .github/workflows/release_build.yml (hunk 7 above)
Comment (martincivan) [reply .github/workflows/release_build.yml]: fixed

**Review by matuszeman — APPROVED** [review]

**Pull request merged**